src/coreclr/jit/emitarm64.cpp

```diff
@@ -6491,7 +6507,8 @@ void emitter::emitIns_R_R_R_I(instruction ins,
                               regNumber   reg3,
                               ssize_t     imm,
                               insOpts     opt /* = INS_OPTS_NONE */,
-                              emitAttr    attrReg2 /* = EA_UNKNOWN */)
+                              emitAttr    attrReg2 /* = EA_UNKNOWN */,
```

Comment:
We are not tracking gc for `reg2` because this is always `EA_UNKNOWN `. This should correctly pass the value.

Reply:
This has been rectified.